src/components/Input.jsx

```diff
@@ -0,0 +1,4 @@
+function Input() {
+  return <></>;
```

Comment:
What is this?)

Reply:
I see in the future, my master sad :
When you develop something you always should think about future usage. The apps tend to change in time, new requirements, new features. If you learn only how to create something for "today" usage you will struggle tomorrow.